erikiado/jp2_online
# Task/panel tutoriales

#### What's this PR do?
Creates the page where all the tutoriales are stored.

#### Where should the reviewer start?
The reviewer should check the model for the Tutorial, and the view base/documentation
Then check the template base/help.txt

#### How should this be manually tested?
The reviewer should login with any type of account, and then go to the tab, for help

#### Any background context you want to provide?
Currently there is a list with all of the tutorials that is created automatically with a migration, all the videos are initialized with a default video that should be changed, after the tutorials are recorded.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
